javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/ExtensionClassLoader.java

```diff
@@ -61,12 +61,6 @@ public static ClassLoader getInstance(ClassLoader parent, File javaagentFile) {
                 System.getenv("OTEL_JAVAAGENT_EXPERIMENTAL_EXTENSIONS")),
             javaagentFile));
 
-    extensions.addAll(
-        parseLocation(
-            System.getProperty(
-                "otel.javaagent.experimental.initializer.jar",
-                System.getenv("OTEL_JAVAAGENT_EXPERIMENTAL_INITIALIZER_JAR")),
-            javaagentFile));
```

Comment:
This property is already handled by the `AgentClassLoader` -- the jar it points to end up being added directly to the agent CL. Loading it as an extension just adds another copy of these classes.